website_google_tag_manager/README.rst

```diff
@@ -1,6 +1,6 @@
-==================
```

Comment:
Why is this file here?

Reply:
Nothings..
But because of that Travis failed.
After change it done

Reply:
Doesn't matter I think. It should go in a separate PR. It can only have been a lint error. In those cases just comment in the PR and say Travis failed because of unrelated module. 

But I let @pedrobaeza say, I don't know every arcane rule of OCA PR's and changes to lint check etc.

Reply:
@smurf-U I doubt this file as it was have any Travis problem. @gdgellatly there are no new rules nor lint checks. Simply proper RST formatting (and people tend to break it), but as I see, it's not the case. If not, please show Travis log to check.

Reply:
@pedrobaeza You can refer below like for Travis failed log
https://travis-ci.org/OCA/website/builds/476770521

Which Give error-

> ************* Module website_google_tag_manager
> website_google_tag_manager/README.rst:3: [E7901(rst-syntax-error), ]  Duplicate implicit target name: "google tag manager".
> ************* Module website_menu_by_user_status

Reply:
@smurf-U Feel free to make a separate PR but that error will not stop this PR merging, it is in an unrelated module.

Tidying up these ones however would be nice.

************* Module website_menu_by_user_status
website_menu_by_user_status/README.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
************* Module website_menu_by_user_status.models.website_menu
website_menu_by_user_status/models/website_menu.py:17: [W8103(translation-field), WebsiteMenu] Translation method _("string") in fields is not necessary.
website_menu_by_user_status/models/website_menu.py:25: [W8103(translation-field), WebsiteMenu] Translation method _("string") in fields is not necessary.
website_menu_by_user_status/models/website_menu.py:31: [W8104(api-one-deprecated), WebsiteMenu._compute_visible] api.one deprecated

Reply:
In any case, this is already fixed here https://github.com/OCA/website/pull/576/files so rebasing PR on HEAD will resolve for you.

Reply:
@gdgellatly will rebase...